### PR TITLE
Feat/boulder list filter

### PR DESCRIPTION
**Pull request by PulexIrritans**
Hi,
I built a filter component/ functionality for the "find" page.
The filter options are generated from the db.
However the filter logic itself is implemented in the front-end. Would have preferred to do it on back-end side too and fetched the "filtered" boulder list but I would have needed to spend more time on it to be able to.
Finally, I think that the front-end logic I have built is somewhat complicated. So if you have any suggestions how I could improve this, this would be very much appreciated.

### CODE REVIEW

File: client/src/components/Filter.jsx [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    sectors: [],
+  });
+  const [holdColorFilter, setHoldColorFilter] = useState('');
+  const [levelFilter, setLevelFilter] = useState('');
+  const [sectorFilter, setSectorFilter] = useState('');
+
+  //   const [filter, setFilter] = useState({
+  //     holdcolorFilter: 'all_colors',
+  //     levelFilter: 'all_levels',
+  //   });
```
Comment (julialoeschel) [inline client/src/components/Filter.jsx:17]: delete code that you dont need

File: client/src/pages/Find.jsx [hunk 2]
```diff
@@ -1,25 +1,67 @@
 import { useEffect, useState } from 'react';
 import Header from '../components/Header';
 import Navigation from '../components/Navigation';
+import Filter from '../components/Filter';
 import BoulderCard from '../components/BoulderCard';
 import styled from 'styled-components';
 const URL = process.env.REACT_APP_URL;
 
-const Find = () => {
+const BoulderList = styled.ul`
```
Comment (julialoeschel) [inline client/src/pages/Find.jsx:9]: I recommend to put Styled Components to the end of the document.

**Review by julialoeschel — COMMENTED** [review]
Great job, I only have a view suggestions 🐜

File: client/src/pages/Find.jsx (hunk 2 above)
Comment (PulexIrritans) [reply client/src/pages/Find.jsx]: Changed

File: client/src/components/Filter.jsx (hunk 1 above)
Comment (PulexIrritans) [reply client/src/components/Filter.jsx]: done

**Comment by PulexIrritans** [comment]
@julialoeschel May I merge now? I have fixed the comments.

**Pull request merged**